app/src/test/java/com/kickstarter/viewmodels/DiscoveryFragmentViewModelTest.java

```diff
@@ -85,11 +85,13 @@ public void testRefresh() {
     // Should emit current fragment's projects.
     this.hasProjects.assertValues(true);
     this.koalaTest.assertValues("Discover List View");
+    this.lakeTest.assertValue("Explore Page Viewed");
```

Comment:
Does this mean you're leaving the koala event the same or am I mis-reading?

Reply:
Yeah they're completely separate `TrackingClient`s. Koala events will be unchanged but they do share properties